<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MalwareIncidentEditor</class>
 <widget class="QDialog" name="MalwareIncidentEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #2d42a8;color: white;</string>
  </property>
  <widget class="QPlainTextEdit" name="IncidentAnalysis">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>550</y>
     <width>1221</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="malwareSave">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>1020</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>1221</width>
     <height>491</height>
    </rect>
   </property>
   <property name="title">
    <string>Incident</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter the Malware Type:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter the Date of When the Malware Was Found:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter the Malware Source: (attachment, website):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>711</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter the Systems Affected By the Malware and associated emails:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Have Any Actions Been Taken Against the Threat?:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>571</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Have the User/Users Computers Been Isolated?:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>What Behaviours Were Exhibited by the Malware?:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter How the Threat Was Detected:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SHA Hash of Malware Artifact if Recovered?:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="malwareType">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>891</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="malwareSource">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>140</y>
      <width>621</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="systemsAffected">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>180</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="threatActions">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>220</y>
      <width>641</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="detectionMethod">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>260</y>
      <width>781</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userIsolation">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>300</y>
      <width>581</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="malwareBehaviour">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>350</y>
      <width>601</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="breachEmail">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>390</y>
      <width>701</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="malwareDateTime">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>100</y>
      <width>621</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>430</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Enter Severity:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ticketStatus">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>440</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Open</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>In Progress</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Closed</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>430</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Ticket Status:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="severity">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>440</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Low</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Medium</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>High</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>522</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>34</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Malware Incident Creation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>1020</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
